packages/kokoas-e2e/cypress/e2e/estimate/shortcuts.cy.ts

```diff
@@ -0,0 +1,155 @@
+
+
+    cy.wait('@getEstimates');
+
+    cy.get('input[name*="costPrice"]').as('costPriceFields');
+    cy.get('input[name*="unitPrice"]').as('unitPriceFields');
+    cy.get('@costPriceFields').eq(0)
+      .as('firstCostPrice');
+    cy.get('@unitPriceFields').eq(0)
+      .as('firstUnitPrice');
+
+    cy.contains('p', /\d+行/)
+      .as('rowCount');
+  });
+  
+  it('行を追加する', () => {
+    
+    cy.get('@rowCount')
+      .invoke('text')
+      .then((oldRowCount) => {
+        const rows = getNumberFromString(oldRowCount);
+
+        cy.get('@firstCostPrice').type('{meta}i');
+        cy.get('@rowCount')
+          .invoke('text')
+          .then((newRowCount) => {
+            const newRows = getNumberFromString(newRowCount);
+            
+            expect(newRows).to.eq(rows + 1);
+          });
+      });
+
+    cy.log('行を追加した後、行の値が初期化されていることを確認します。');
+    cy.get('@costPriceFields').eq(1)
+      .should('have.value', '0');
+  });
+
+  it('行を複製する', () => {
+    cy.get('@rowCount')
+      .invoke('text')
+      .then((oldRowCount) => {
+        const rows = getNumberFromString(oldRowCount);
+        const randomCostPrice = Math.floor(Math.random() * 8000) + 1000;
+        const maxRange = Math.floor(randomCostPrice * 0.9);
+        const randomUnitPrice = Math.floor(Math.random() * maxRange) + randomCostPrice;
+
+        cy.get('@firstCostPrice').type(`${randomCostPrice.toString()}`);
+        cy.get('@firstUnitPrice').type(`${randomUnitPrice.toString()}{meta}{shift}i`);
+
+        cy.log('行を複製した後、行の値が複製元の行と同じになっていることを確認します。');
+        cy.get('@costPriceFields').eq(1)
+          .should('have.value', randomCostPrice.toLocaleString());
+        cy.get('@unitPriceFields').eq(1)
+          .should('have.value', randomUnitPrice.toLocaleString());
+
+        cy.get('@rowCount')
+          .invoke('text')
+          .then((newRowCount) => {
+            const newRows = getNumberFromString(newRowCount);
+            expect(newRows).to.eq(rows + 1);
+          });
+      });
+  });
+
+  it('行を削除する', () => {
+      
+    cy.get('@rowCount')
+      .invoke('text')
+      .should('eq', `${testRowCount}行`);
+
+    cy.log('１行目を削除したら、フォーカスは次の行に移動する');
+    cy.get('@firstCostPrice').type('{meta}{del}', { scrollBehavior: 'center' });
+    cy.focused().should('have.attr', 'name', 'items.0.costPrice');
+
+    cy.log('3行目を削除したら、フォーカスは次の行に移動する');
+    cy.get('@costPriceFields')
+      .eq(2)
+      .type('{meta}{del}', { scrollBehavior: 'center' });
+    cy.focused().should('have.attr', 'name', 'items.2.costPrice');
+
+    cy.log('最終行を削除したら、フォーカスは前の行に移動する');
+    cy.get('@costPriceFields').eq(2)
+      .type('{meta}{del}', { scrollBehavior: 'center' });
+    cy.focused().should('have.attr', 'name', 'items.1.costPrice');
+
+    cy.get('@costPriceFields').eq(1)
+      .type('{meta}{del}', { scrollBehavior: 'center' });
+    cy.focused().should('have.attr', 'name', 'items.0.costPrice');
+
+    cy.log('最終行は一つしかないなら、フォーカスは仮想業の行に移動する');
```

Comment:
```suggestion
    cy.log('最終行は一つしかないなら、フォーカスは仮想業の行に移動する');
```

```最終行は一つしかないなら、フォーカスは仮想行に移動する```
の誤記かと思います。
ご確認をお願いします。